_posts/2023-10-03-flux-sched-0.29.0.md

```diff
@@ -0,0 +1,44 @@
+---
+title: flux-sched v0.29.0
+date: 2023-10-03 00:00:00
+author: "flux-framework"
+categories: 'release'
+version: 0.29.0
+download: https://github.com/flux-framework/flux-sched/releases/download/v0.29.0/flux-sched-0.29.0.tar.gz
+---
+
+Download from GitHub [here]({{ page.download }})
+
+# Release Notes
+
+flux-sched version 0.29.0 - 2023-10-03
+--------------------------------------
+
+### New Features
+
+ * Add first nodex policy ([#1055](https://github.com/flux-framework/flux-sched/issues/1055))
+ * Enable Fluxion resource graph elasticity ([#989](https://github.com/flux-framework/flux-sched/issues/989))
+ * add delete subgraph functionality ([#1053](https://github.com/flux-framework/flux-sched/issues/1053))
+ * Update branch of golang release ([#1062](https://github.com/flux-framework/flux-sched/issues/1062))
```

Comment:
Wow! Elasticity, delete subgraph, and golang?! nice!! I just don't know what a "nodex" is otherwise I'd include that one too. :laughing: